customizations/generic/templates/frame.haml

```diff
@@ -0,0 +1,386 @@
+      %link{type:"image/ico", rel:"shortcut icon", href:"{{ STATIC_URL }}{{ brand.favico }}"}
+    -else
+      %link{type:"image/ico", rel:"shortcut icon", href:"{{ STATIC_URL }}images/favicon.ico"}
+
+    -load smartmin sms
+    -load i18n
+
+    -block styles
+      -compress css
+        %link{type:'text/css', rel:'stylesheet', href:'{{ STATIC_URL }}bower/bootstrap-css/css/bootstrap.css',  media:'all' }
+        %link{type:'text/css', rel:'stylesheet', href:'{{ STATIC_URL }}bower/animate.css/animate.css',  media:'all' }
+
+      -compress css
+        %link{type:'text/css', rel:'stylesheet', href:'{{ STATIC_URL }}bower/bootstrap-css/css/bootstrap-responsive.css',
+              media:'all and (min-width: 1200px), all and (max-width: 767px), all and (max-width: 480px), all and (min-width: 980px)' }
+
+      -compress css
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}bower/toastr/toastr.css", media:'all'}
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}bower/select2/select2.css", media:'all'}
+        %link{type:'text/css', rel:'stylesheet', href:'{{ STATIC_URL }}bower/At.js/dist/css/jquery.atwho.css', media:'all'}
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}bower/jquery-ui/themes/redmond/jquery-ui.css", media:'all'}
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}bower/jquery-ui/themes/redmond/jquery.ui.theme.css", media:'all'}
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}bower/featherlight/src/featherlight.css", media:'all'}
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}bower/video.js/dist/video-js.css", media:'all'}
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}bower/videojs-vjsdownload/dist/videojs-vjsdownload.css", media:'all'}
+
+      -compress css
+        %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}fonts/temba/style.css", media:'all'}
+        %link{type:'text/less', rel:'stylesheet', href:'{{STATIC_URL}}less/print.less', media:'print'}
+
+      -compress css
+
+        {% lessblock %}
+          :plain
+            @import "formax.less";
+            @import "style.less";
+
+            // any brand styling tweaks
+            @import (optional, reference) "../brands/style.less";
+            @import (optional) "../brands/{{brand.slug}}/less/style.less";
+
+        {% endlessblock %}
+
+        // any additional brand styling such as fonts, etc
+        -for style in brand.styles
+          -if 'less' in style
+            %link{type:'text/less', rel:'stylesheet', href:'{{STATIC_URL}}{{style}}', media:'all'}
+          -else
+            %link{type:'text/css', rel:'stylesheet', href:"{{ STATIC_URL }}{{style}}", media:'all'}
+
+      -block extra-less
+      -block extra-style
+
+  -for incl in brand.head_includes
+    -include incl
+
+  <!--[if lt IE 9]>
+    %script{src:"//html5shim.googlecode.com/svn/trunk/html5.js"}
+  <![endif]-->
```

Comment:
lines 168 through 173 should be indented 2 more spaces as they will render between the </head><body> tags otherwise.

Reply:
@andrewkcarter @baracudda lets merge this and address this in another ticket, since the PR is holding up a lot of templates and code.